Cargo.toml

```diff
@@ -1,61 +1,20 @@
-[package]
+[workspace.package]
 name = "autometrics"
 version = "0.2.4"
 edition = "2021"
 authors = ["Fiberplane <info@fiberplane.com>", "Evan Schwartz <3262610+emschwartz@users.noreply.github.com>"]
 description = "Easily add metrics to your system -- and actually understand them using automatically customized Prometheus queries"
 documentation = "https://docs.rs/autometrics"
-readme = "README.md"
 repository = "https://github.com/fiberplane/autometrics-rs"
 license = "MIT OR Apache-2.0"
 keywords = ["metrics", "prometheus", "opentelemetry"]
 categories = ["development-tools::profiling"]
 
-[features]
-default = ["opentelemetry"]
-metrics = ["dep:metrics"]
-opentelemetry = ["opentelemetry_api"]
-prometheus = ["dep:prometheus", "once_cell"]
-prometheus-exporter = [
-  "metrics-exporter-prometheus",
-  "once_cell",
-  "opentelemetry-prometheus",
-  "opentelemetry_sdk",
-  "prometheus"
+[workspace]
+default-members = ["autometrics", "autometrics-macros"]
+members = [
+  "autometrics",
+  "autometrics-macros",
+  "examples/*"
 ]
-
-[dependencies]
-autometrics-macros = { version = "0.2.0", path = "./macros" }
-const_format = { version = "0.2", features = ["rust_1_51"] }
-
-# Used for opentelemetry feature
-opentelemetry_api = { version = "0.18", default-features = false, features = ["metrics"], optional = true }
-
-# Use for metrics feature
-metrics = { version = "0.20", default-features = false, optional = true }
-
-# Used for prometheus-exporter feature
-metrics-exporter-prometheus = { version = "0.11", default-features = false, optional = true }
-once_cell = { version = "1.17", optional = true }
-opentelemetry-prometheus = { version = "0.11", optional = true }
-opentelemetry_sdk = { version = "0.18", default-features = false, features = ["metrics"], optional = true }
-prometheus = { version = "0.13", default-features = false, optional = true }
-
-[dev-dependencies]
-axum = "0.6.4"
-opentelemetry-prometheus = { version = "0.11" }
-rand = "0.8.5"
-reqwest = { version = "0.11.14", features = ["json"] }
-serde = { version = "1.0.152", features = ["derive"] }
-strum = { version = "0.24.1", features = ["derive"] }
-thiserror = "1.0.38"
-tokio = { version = "1.24.2", features = ["full"] }
-
-[[example]]
-name = "axum"
-required-features = ["prometheus-exporter"]
-path = "examples/axum.rs"
-
-[package.metadata.docs.rs]
-all-features = true
-rustdoc-args = ["--cfg", "docsrs"]
+exclude = ["examples/data", "examples/target"]
```

Comment:
maybe add these two as `[[example]]` keys? https://doc.rust-lang.org/cargo/reference/cargo-targets.html#examples

Reply:
Those are actually the two things that aren't examples :)

Reply:
i cant read my bad